.gitignore

```diff
@@ -1,3 +1,15 @@
 Calculator/.vscode
 CRUD/.vscode
-SPA/.vscode
\ No newline at end of file
+SPA/.vscode
```

Comment:
use wildcard ** for node_modules as **/node_modules and see if that works, instead of declaring all separately.

Reply:
**/.vscode
**/node_modules
**/.env

I have written like above but  still .env files are uploaded to remote repository